_includes/head.html

```diff
@@ -3,6 +3,7 @@
 <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
 <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
+<link rel="shortcut icon" type="image/x-icon" href="https://www.rust-lang.org/favicon.ico">
```

Comment:
I'd probably leave the favicon blank until there's a decent logo for hyper.

Reply:
Fair enough. I wasn't sure your stance on it.